repos/system_upgrade/common/actors/udev/udevadminfo/libraries/udevadminfo.py

```diff
@@ -0,0 +1,7 @@
+from leapp.libraries.stdlib import api, run
+from leapp.models import UdevAdmInfoData
+
+
+def process():
+    out = run(['udevadm', 'info', '-e'])['stdout']
```

Comment:
yes, udevadm is always available. it's part of systemd-udev library. Some error catching can be added, however, considering this command mast always work otherwise the system is broken basically.